py/py2/ocGraph.py

```diff
@@ -16,48 +16,52 @@ class StripDrawer(igraph.drawing.shapes.ShapeDrawer):
 
     @staticmethod
     def draw_path(ctx, center_x, center_y, width, height=20):
-        ctx.rectangle(center_x - width/2, center_y - height/2, width, height)
+        ctx.rectangle(
+            center_x - width / 2, center_y - height / 2, width, height
+        )
 
     @staticmethod
-    def intersection_point(center_x, center_y, source_x, source_y, width, height=20):
+    def intersection_point(
+        center_x, center_y, source_x, source_y, width, height=20
```

Comment:
Sticking with 79 chars/line is kind of strict and may not be optimal for today's python. If we are going with 79, I think something like this would be more readable:
`def intersection_point(center_x, center_y, source_x,
                                     source_y, width, height=20):`

Reply:
I can increase it to 88 as per black's default's, though 79 is the Python standard and we generally want to avoid super long lines to ease reading, especially when side by side in a diff or something similar.